File: Python/variabile.py
# variabila = zona din memoria unui calculator care tine date
# variabila = cutiuta in care punem valori

# am declarat (i-am dat un nume - marca) si initializat variabila marca
marca_masina = 'Volvo'
model_masina = 'XC 60'

# nu putem sa punem spatiu in numele variabilei
# o variabila incepe cu litera mica

# loosely typed - nu trebuie sa specificam tipurile

print(f'Am cumparat o masina marca : {marca_masina}')
print(f'Este modelul : {model_masina}')

# suprascriere
model ='XC 60 facelift'
print(f'Am cumparat o masina marca : {marca_masina}')
print(f'Este modelul : {model_masina}')

# concatenare de stringuri
nume = 'Simion'
prenume = 'Tabeea'
varsta = 27
print(nume + ' ' + prenume)
print(f'{nume} {prenume} si am varsta de {varsta} de ani.')

File: Java/Firststeps/Firststeps/src/Variabile.java
public class Variabile {
    public static void main(String[] args) {
        // declarare si initializare
        String marcaMasina = "Volvo";
        String modelMasina = "XC 60";

        // strongly typed - trebuie sa specificam tipul de date al variabilelor cu care lucram

        System.out.println("Am cumparat o masina marca: " + marcaMasina);
        System.out.println("Este modelul: " + modelMasina);

        // suprascrierea
        modelMasina = "XC 60 facelift"; // nu mai este nevoie sa ii spun tipul de date - este deja declarat

        System.out.println("Am cumparat o masina marca: " + marcaMasina);
        System.out.println("Este modelul: " + modelMasina);

        // declarare
        String nume;
        String prenume;
        // initializare
        nume = "Simion";
        prenume = "Tabeea";
        int varsta = 27;
        // concatenare de stringuri
        System.out.println(nume + " " + prenume + "cu varsta de " + varsta + " de ani");

    }
}
